### PR TITLE
Update DeveloperGuide.md

### PR DESCRIPTION
**Pull request by D-Limiter**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -408,7 +408,26 @@ in `CommandResult`, informing `UI` to update similarly to what we did in **help*
...
+
+The stats command provides insights into a user's learning journey within Flashlingo. By visualizing progress and areas of improvement, this feature aims to motivate users and enhance their learning experience.
+
+#### Implementation
+
+The stats command accesses the stored number of words remembered from the last session, accesses them and retrun the result as a success rate to help inspire the user to continue learning.
+
+The execute method accesses the numberOfWords from the model.
+
+To be added: Making th success rate more relatable by remembering data(storing) from the last session.
```
Comment (itsNatTan) [inline docs/DeveloperGuide.md:423]: Typo on this line. Making the* success rate

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -408,7 +408,26 @@ in `CommandResult`, informing `UI` to update similarly to what we did in **help*
...
 
+### Learning Statistics
 
+#### **Feature Overview**
+
+The stats command provides insights into a user's learning journey within Flashlingo. By visualizing progress and areas of improvement, this feature aims to motivate users and enhance their learning experience.
+
+#### Implementation
+
+The stats command accesses the stored number of words remembered from the last session, accesses them and retrun the result as a success rate to help inspire the user to continue learning.
```
Comment (itsNatTan) [inline docs/DeveloperGuide.md:419]: Typo. accesses them and returns* the result as a success...

**Review by itsNatTan — COMMENTED** [review]
Fix typo

**Review by WangCheng0116 — CHANGES_REQUESTED** [review]
* Where is the improvement in `list`? Our application doesn't work the same as before. Need to change accordingly. For example, after `list`, translation will not be shown unless reveal button is pressed.  
* What's the expected output of `stats`
* Expected Output of `review`

**Pull request closed without merging**